CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing to License List Data
+
+Technical questions or issues can be sent to the [SPDX technical team mailing list](mailto:spdx-tech@lists.spdx.org).
+Questions or issues on the license content can be sent to the [SPDX legal team mailing list](mailto:spdx-legal@lists.spdx.org).
+
+Note that the content of this repository is generated by the [SPDX tools](http://github.com/spdx/tools) from the [license list source](https://github.com/spdx/license-list-XML).
+
+As far as the *content* of the license information is concerned, see the [license-list-XML Contributing](https://github.com/spdx/license-list-XML/blob/master/CONTRIBUTING.md) document for information on how to report issues, request new licenses, or contribute changes.
+
+As far as the output *format* or the *generation* of the output files is concerned, any [issues](https://github.com/spdx/tools/issues) or [pull requests](https://github.com/spdx/tools/pulls) should be logged at the [SPDX tools](http://github.com/spdx/tools) project.
```

Comment:
Do these need to be updated to point at https://github.com/spdx/LicenseListPublisher?

Reply:
Good catch!

I'll update